.github/workflows/test-linux.yml

```diff
@@ -1,10 +1,6 @@
 name: Build (Linux)
 
 on:
-  push:
-    branches:
-      - master
-      - 1.4.x
   pull_request:
     branches:
       - master
```

Comment:
what about `pull_request`?

Reply:
mmh yeah we can double check the intention with @acogoluegnes but this translates to "run this ci process for pull_requests that target the given branches". the `pull_request` trigger, without a `types` clause, means this ci run when PRs are `opened`, `reopened` and `synchronize`d (which should mean it runs every time new commits are added)